packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -200,7 +200,7 @@ const schema: SchemaType<"Posts"> = {
     permissions: {
       canRead: ['guests'],
       canUpdate: ['members', 'sunshineRegiment', 'admins'],
-      canCreate: [userHasModerationGuidelines]
+      canCreate: ['members', 'sunshineRegiment', 'admins'],
```

Comment:
`userHasModerationGuidelines` checks `hasAuthorModeration`, which is a feature flag the EA forum has disabled. So this probably needs to be `hasAuthorModeration ? [] : ['members', 'sunshineRegiment', 'admins']`.

Reply:
There are two things that `canCreate` can do. It can affect what's on the creation form, but there isn't a post-creation form anymore, just a redirector. And it can cause create-mutations to fail, which, I don't think is what EA forum wants under any scenario.